### PR TITLE
Add default file to unpkg and jsdelivr

### PR DESCRIPTION
**Pull request by GMartigny**

### CODE REVIEW

**Comment by ScottyFillups** [comment]
Thanks :)

File: README.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Yet another **Javascript noise library**. Demonstrations [here](http://scottyfil
 npm install tumult --save
 ```
 
-The built files are also available on `unpkg`:
+The built files are also available on `unpkg` and `jsdelivr`:
 
 
 ```html
-<script src="https://unpkg.com/tumult/dist/tumult.min.js"></script>
+<script src="https://unpkg.com/tumult"></script>
```
Comment (ScottyFillups) [inline README.md]: Will this work? Keep in mind https://unpkg.com/tumult points to the source code that uses CommonJS imports:

```js
// Response from visiting https://unpkg.com/tumult
'use strict'

module.exports = require('./lib')
```

We probably want 

```html
<script src="https://cdn.jsdelivr.net/npm/tumult/dist/tumult.min.js"></script>
<script src="https://unpkg.com/tumult/dist/tumult.min.js"></script>
```

Can you please confirm? Please update your PR if necessary and I'll merge it 😄

Comment (GMartigny) [reply README.md]: That's the point of the change in `package.json`. With that change, https://unpkg.com/tumult will point to the path specified, being `"dist/tumult.min.js"`.

Comment (GMartigny) [reply README.md]: See https://unpkg.com/pencil.js with https://github.com/pencil-js/pencil.js/blob/master/modules/pencil.js/package.json

Comment (ScottyFillups) [reply README.md]: Wow that's pretty cool. Thanks for showing me that, didn't know it existed 😮 

I'll merge it! Thank you 😄

**Pull request merged**